src/main/java/org/spongepowered/api/entity/attribute/Attribute.java

```diff
@@ -0,0 +1,94 @@
+package org.spongepowered.api.entity.attribute;
+
+import org.spongepowered.api.entity.attribute.operation.AttributeOperation;
+import org.spongepowered.api.entity.attribute.type.AttributeType;
+
+import java.util.Collection;
+import java.util.Optional;
+import java.util.UUID;
+
+/**
+ * An attribute instance.
```

Comment:
Clarify? Is it just a container for some values? What does it represent? Who can own one?